ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -49,53 +51,61 @@ struct MSFListCellView: View {
             }
         }, label: {
             HStack(spacing: 0) {
+                let defaultIconSize = tokens.iconDefaultSize
```

Comment:
isn't the icon size in relation with to the size of the list?

Reply:
Nope, there are currently 2 icon sizes, default and large.

Reply:
And we now need to add at least the 16x16 size in addition to these 2.